MediaBrowser.MediaEncoding/Subtitles/SubtitleEncoder.cs

```diff
@@ -542,7 +614,7 @@ private async Task ExtractAllExtractableSubtitlesInternal(
                 var outputCodec = IsCodecCopyable(subtitleStream.Codec) ? "copy" : "srt";
                 var streamIndex = EncodingHelper.FindIndex(mediaSource.MediaStreams, subtitleStream);
 
-                if (streamIndex == -1)
+                if (streamIndex == -1 || subtitleStream.Path.EndsWith(".mks", StringComparison.OrdinalIgnoreCase))
```

Comment:
I'd split this `if` and use a different message for `.mks`.

Reply:
You're right. I separated those conditions and changed added a different debug message for the case that the sub is part of an MKS file.